src/main/java/org/intellij/sdk/codesync/clients/WebSocketClientEndpoint.java

```diff
@@ -0,0 +1,65 @@
+package org.intellij.sdk.codesync.clients;
```

Comment:
Are we using this?

Reply:
yes, it being used as the underlying websocket implementation.

Reply:
great then 👍